src/trigrams.rs

```diff
@@ -1,6 +1,6 @@
 use std::collections::HashMap;
 
-pub fn get_trigrams_with_positions(text : &String) -> HashMap<String, u32> {
+pub fn get_trigrams_with_positions(text : &str) -> HashMap<String, u32> {
```

Comment:
@Dr-Emann I like to ask (maybe you can help with this). What is the most convenient and efficient way to pass text to query? https://github.com/greyblake/whatlang-rs/blob/master/src/query.rs

Currently is accepts text as `&String`, but it feels a little bit obtrusive  to pass `&text` instead of `text`.